script/setup.sh

```diff
@@ -65,6 +65,11 @@ case "$(uname -s)" in
 	# xpath for xml parsing
 	URL_NASR_CURRENT=$(curl -s $URL_NASR_EDITION | xpath -q -e 'string(//productSet/edition/product/@url)')
      ;;
+
+   MINGW*)
```

Comment:
Does this require or assume a specific version of MINGW? Or is the wildcard sufficient for all versions?

Reply:
The wildcard should catch both 32-bit and 64-bit versions of MINGW, as well as all version numbers.